roles/klaytn_node/tasks/main.yml

```diff
@@ -118,3 +127,10 @@
   include_tasks: "gen_nodekey.yml"
   when:
     - ansible_os_family != "Windows"
+    - klaytn_service_type != "grafana"
```

Comment:
My opinion is to make new grafana role, what do you think about that?

Reply:
Like I added a new role for configuring bridge? A new role like `klaytn_grafana`?

That is good, except the newly added role still needs the list of all nodes' IP for the `prometheus.yml` file. Those IPs are currently fetched via the `inventory` file however using another role would require a new way to get the list of nodes' IPs.

But still, that is a good suggestion. I will look for a nice way to solve above issue. Thank you for the comment.

Reply:
Done de8fbcc
Instead of using the `inventory` file for the Klaytn node list, added a variable `klaytn_node_list` in playbook.
This is because we only need the IP addresses, not the `user`. Specifying the IP and user in the `inventory` file may confuse the user as if we need SSH connection to Klaytn nodes.

Reply:
On a second thought, how about specifying the `klaytn_node_list` in the `inventory` file's host `GRAFANA`?
For example,
```
[Grafana]
GRAFANA ansible_host=1.2.3.4 ansible_user=centos klaytn_node_list=["5.6.7.8", "5.6.7.9"]
```

Reply:
Moved `klaytn_node_list` to `inventory` file in 5cb62df.

Reply:
Marking this resolved.